api/Makefile

```diff
@@ -4,6 +4,11 @@
 
 APP_NAME := grants-api
 
+# Colors for output, can be used as:
```

Comment:
Should these colors be applied to the other checks?

Reply:
I think most of our other formatter/linter/test utils do a good job of printing out proper error messages automatically in a clear way (eg. mypy might tell you specifically what line and problem there is)

In this case, it wasn't as clear what the issue was as it just ended:
```
make: *** [Makefile:168: db-check-migrations] Error 255
Error: Process completed with exit code 2.
```
which isn't terribly helpful - the lines above that say what migrations it detected, but at first glance you might not know what you're looking at.

If any of the others aren't clear to someone, happy to add messaging to them, but that would be separate